app/src/app/[lng]/admin/page.tsx

```diff
@@ -117,6 +131,26 @@ const AdminPage = ({ params: { lng } }: { params: { lng: string } }) => {
     }
   };
 
+  const handleChangeOrganizationStatus = async (
+    activeStatus: boolean,
+    organizationId: string,
+  ) => {
+    const updateStatusResponse = updateOrganizationActiveStatus({
+      activeStatus,
+      organizationId,
+    });
+
+    toaster.promise(updateStatusResponse, {
+      success: {
+        title: t("organization-status-updated"),
+      },
+      error: {
+        title: t("error-occurred"),
+      },
+      loading: { title: t("upddating-status") },
```

Comment:
Typo in translation string name
```suggestion
      loading: { title: t("updating-status") },
```